docs/ja/changelog.md

```
@@ -1094,3 +1102,5 @@ outline: deep
 - ReDoS（正規表現サービス拒否）防止
 - 全 API エンドポイントの入力検証
 
+[unreleased]: https://github.com/geolonia/geonicdb/compare/v0.1.0...HEAD
+[0.1.0]: https://github.com/geolonia/geonicdb/releases/tag/v0.1.0
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "0.1.0"

(MD053, link-image-reference-definitions)